### DISCUSSION TITLE
Can't find package.json

### DISCUSSION THREAD

**Discussion by binarykitchen** [web-infra-dev/rslib#359] [General]
When you define an external rslib config file with the `-c` parameter, it can't find the package.json anymore. For example:

```
rslib build -c ./etc/rsbuild/config.ts
```

It will complain that it can't find the package.json from the root folder.

**Comment by fi3ework** [comment]
I believe it would be a bug, and we didn't cover that use case in out test. let's track that in https://github.com/web-infra-dev/rslib/issues/361.
